src/backend/storeManagers/gog/games.ts

```diff
@@ -882,3 +894,157 @@ export function isGameAvailable(appName: string) {
   }
   return false
 }
+
+async function postPlaytimeSession({
+  appName,
+  session_date,
+  time
+}: GOGSessionSyncQueueItem) {
+  const userData: UserData | undefined = configStore.get_nodefault('userData')
+
+  const credentials = await GOGUser.getCredentials().catch(() => null)
+
+  if (!userData || !credentials) {
```

Comment:
maybe we can split this check in 2 and move the `if (!userData)` above? if we already know userData is undefined there's no need to try to get the credentials

and it also makes debugging easier since you can have 2 different messages, now if you check 2 variables and print a shared message in the logs then you would have to reproduce this issue to know if the problem was either missing userData or failure to get the credentials